tests/test_esm_runscripts/test_filedicts.py

```diff
@@ -164,28 +181,58 @@ def test_mv(fs):
     """Tests for mv"""
     dummy_config = """
     general:
-        thisrun_work_dir: "/work/ollie/pgierz/some_exp/run_20010101-20010101/work"
         exp_dir: "/work/ollie/pgierz/some_exp"
         thisrun_dir: "/work/ollie/pgierz/some_exp/run_20010101-20010101"
     computer:
         pool_dir: "/work/ollie/pool"
     echam:
         files:
             jan_surf:
-                name_in_pool: T63CORE2_jan_surf.nc
-                path_in_pool: ECHAM/T63/
+                name_in_computer: T63CORE2_jan_surf.nc
+                path_in_computer: /work/ollie/pool/ECHAM/T63/
                 name_in_work: unit.24
                 path_in_work: .
-        thisrun_work_dir: /work/ollie/mandresm/awiesm/run_20010101-20010101/work/
+        thisrun_work_dir: "/work/ollie/pgierz/some_exp/run_20010101-20010101/work"
     """
     config = yaml.safe_load(dummy_config)
     fs.create_file("/work/ollie/pool/ECHAM/T63/T63CORE2_jan_surf.nc")
     fs.create_dir("/work/ollie/pgierz/some_exp/run_20010101-20010101/work")
     assert os.path.exists("/work/ollie/pool/ECHAM/T63/T63CORE2_jan_surf.nc")
     sim_file = esm_runscripts.filedicts.SimulationFile(config, "echam.files.jan_surf")
-    sim_file.mv("pool", "work")
+    sim_file.mv("computer", "work")
     assert not os.path.exists("/work/ollie/pool/ECHAM/T63/T63CORE2_jan_surf.nc")
     assert os.path.exists(
         "/work/ollie/pgierz/some_exp/run_20010101-20010101/work/unit.24"
     )
 
+
+def test_check_path_in_computer_is_abs(fs):
+    """
+    Tests that ``esm_parser.user_error`` is use when the ``path_in_computer``
+    is not absolute
+    """
+
+    dummy_config = """
+    general:
+        thisrun_dir: "/work/ollie/pgierz/some_exp/run_20010101-20010101"
+        exp_dir: "/work/ollie/pgierz/some_exp"
+    echam:
+        files:
+            jan_surf:
+                name_in_computer: T63CORE2_jan_surf.nc
+                name_in_work: unit.24
+                path_in_computer: pool/ECHAM/T63
+        thisrun_work_dir: "/work/ollie/pgierz/some_exp/run_20010101-20010101/work"
+    """
+    config = yaml.safe_load(dummy_config)
+
+    # Captures output (i.e. the user-friendly error)
+    with Capturing() as output:
+        try:
+            sim_file = esm_runscripts.filedicts.SimulationFile(config, "echam.files.jan_surf")
+        except SystemExit as e:
+            error = e
+
+    # error needs to occur as the path is not absolute
+    assert isinstance(error, SystemExit)
+    assert any(["ERROR: File Dictionaries" in line for line in output])
```

Comment:
I thinks a single assert is enough. Probably, nothing will get printed after the first assert.